desktop/core/src/desktop/js/apps/admin/Overview/Overview.scss

```diff
@@ -0,0 +1,50 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import '../../../components/styles/variables';
+
+.antd.cuix {
+  .hue-overview-component {
+    background-color: $fluidx-gray-100;
+    padding: 24px;
+
+    .config__address-value {
+      color: $fluidx-blue-600;
+      padding: 2px 4px;
+      background-color: #f7f7f9;
```

Comment:
Do we have Cloudera colors for these values? If not give them a named var here and reuse or try and find predfined colors that work.